### ISSUE TITLE
How to remove the copyright box?

### ISSUE THREAD

**Issue by mcanini** [borisveytsman/acmart#169]
For a submission, I don't need any info on copyright. However using \setcopyright{none} still takes up space. Is there a way to remove the copyright box?

**Comment by borisveytsman** [comment]
I am not sure I understand the request.  Attached is the result of `\setcopyrigth{none}`.  What do you want me to delete?
[tmp.pdf](https://github.com/borisveytsman/acmart/files/1045443/tmp.pdf)

**Comment by mcanini** [comment]
I am using the sigconf flavor. I suppose what I want it to disable the \footnotetextcopyrightpermission block of code at https://github.com/borisveytsman/acmart/blob/master/acmart.dtx#L4236
When I comment it out, there is no space taken up by the conference info.
Thanks.

**Comment by Craig-Rodkin** [comment]
Hello, ACM makes the authorverion and authordraft versions available to you. As per ACM SIG requirement, we were tasked to include space for the ACM copyright block for all potential submissions.

**Comment by dbeyer** [comment]
This question actually raises an interesting issue that should be reflected in the formatting requirements in each call for paper that is based on the ACM style: The guideline should be that for a submission, the authors should use the copyright statement that they intend to use later for the camera-ready version.

If you (for a submission) try to remove the space taken by the legal text, or if you even try to reduce it with \setcopyright{none}, then you later, after the review, have to remove a considerable portion of the text that the reviewers wanted to publish. It is like circumventing the page limit a tiny little bit. ;-)

In other words, one should never use \setcopyright{none} for a submission.
(I guess this option exists for cases when you use the ACM style for a publication on arXiv or so.)

**Comment by mcanini** [comment]
That might be true for certain venues. But many of the conferences I submit to actually use different formatting guidelines: submissions use 10pt fonts while final versions it is 9pt. So I'm not concerned with removing that space for copyright boxes...

**Comment by Craig-Rodkin** [comment]
ACM’s templates have been used in different ways over the past decade. (I’ve even seen other publishers take the ACM template and reuse in their publications as well.)

I believe the main point is that we have tried to be as flexible for authors and organizers as possible in their various use cases of our templates. Whether publishing within an ACM venue, posting to a repository, or just an internal technical report, ACM, along with the group of volunteers who worked on this project have come up with a very good, flexible product which I hope works for our authors

Regards,

-Craig

Craig Rodkin
Publications Operations Manager
Association for Computing Machinery
rodkin@hq.acm.org<mailto:rodkin@hq.acm.org>
http://member.acm.org/~rodkin

Explore Association for Computing Machinery's AUTHOR-IZER SERVICE:  http://www.acm.org/publications/acm-author-izer-service<http://www.acm.org/publications/acm-author-izer-service>

**Comment by PeterSewell** [comment]
I'd like to echo this request.  It is common in our field to distribute preprints of submitted papers.  For those, there is a lot of extraneous noise that does not seem easy to turn off.  This is using "acmsmall" for PACMPL; I've just had to manually hack the class file to remove:
- the copyright notice, DOI, price, and first-page footnote hrule
- the article number and colon for each page number
- the black folioblob on the first page

thanks,
Peter

**Comment by Craig-Rodkin** [comment]
I’ll put this forth within our annual review process, but the primary use of ACM templates are to publish articles within an ACM publication; there is a need to hold the rights block space so that page layout and counts are set before an author submits to be published.

I would also mention that the following versions already exist:

Option Default Meaning
review false A review version: lines are numbered, hyperlinks are
colored
screen false A screen version: hyperlinks are colored
natbib true Whether to use natbib package (see Section 2.10)
anonymous false Whether to make author(s) anonymous
authorversion false Whether to generate a special version for authors’ personal
use or posting (see Section 2.3)
timestamp false Whether to put a time stamp in the footer of each page
authordraft false Whether author’s draft mode is enabled

Please see page 7 of the documentation
Regards,

-Craig

Craig Rodkin
Publications Operations Manager
Association for Computing Machinery
rodkin@hq.acm.org<mailto:rodkin@hq.acm.org>
http://member.acm.org/~rodkin

Explore Association for Computing Machinery's AUTHOR-IZER SERVICE:  http://www.acm.org/publications/acm-author-izer-service<http://www.acm.org/publications/acm-author-izer-service>

**Comment by PeterSewell** [comment]
Thanks for the response.

On 5 July 2017 at 20:28, Craig-Rodkin <notifications@github.com> wrote:

> I’ll put this forth within our annual review process, but the primary use
> of ACM templates are to publish articles within an ACM publication; there
> is a need to hold the rights block space so that page layout and counts are
> set before an author submits to be published.
>

It's typically the case for us that the final version will differ in many
respects, and sometimes be subject to a different page limit from the first
submission.  In any case, distributing versions of papers prior to
publication is an essential part of the scientific process, and hence also
part of the "publish articles within an ACM publication" process.
Supporting authors flexibly would be a good goal.


>
> I would also mention that the following versions already exist:
>
> Option Default Meaning
> review false A review version: lines are numbered, hyperlinks are
> colored
> screen false A screen version: hyperlinks are colored
> natbib true Whether to use natbib package (see Section 2.10)
> anonymous false Whether to make author(s) anonymous
> authorversion false Whether to generate a special version for authors’
> personal
> use or posting (see Section 2.3)
> timestamp false Whether to put a time stamp in the footer of each page
> authordraft false Whether author’s draft mode is enabled
>

I'm aware of those, thanks.

Peter

**Comment by Craig-Rodkin** [comment]
>> Supporting authors flexibly would be a good goal.

Within reason (and budget), we are trying to be as flexible as possible. This is why the existing variants exist.

The rights block space inclusion was deemed important to leave in by the volunteers within the TeX template group and will be addressed with the other requested changes at the end of this year.

Regards,

-Craig

Craig Rodkin
Publications Operations Manager
Association for Computing Machinery
rodkin@hq.acm.org<mailto:rodkin@hq.acm.org>
http://member.acm.org/~rodkin

Explore Association for Computing Machinery's AUTHOR-IZER SERVICE:  http://www.acm.org/publications/acm-author-izer-service<http://www.acm.org/publications/acm-author-izer-service>

**Comment by MatthewFluet** [comment]
I'm note sure which TeX template group volunteers @Craig-Rodkin is referring to, but I've advocated for options to set a fixed-size for and to suppress the copyright space: #8 and #10.  The decision to close those issues seems to have come from the ACM editors.

**Comment by samth** [comment]
Consider this another user who would find this option valuable. Perhaps we need to bring this up with the PACMPL editorial board, though.

**Comment by borisveytsman** [comment]
I agree:  this is the decision for editors to make rather than for me...

**Comment by PeterSewell** [comment]
Having had to spend quite some time again last week hacking acmart to
produce a clean version of a not-yet-submitted paper for distribution (on
my web page and via arxiv), I'd like to re-emphasise how desirable this
would be.  I think it would be fairly easy for a good latex hacker (which I
am not): it's just a matter of adding a "clean" or "plain" option that
turns things off.   In the acmsmall variant used for JACM, I had to:

- replace the copyright footnote (\copyright\ \@copyrightyear\
\@copyrightowner) with just the date
- turn off the journal name and DOI
- turn off the ACM reference format
- fix the headers and footers to turn off the article number and :, and the
journal name, volume, etc.
- turn off the "This is the author's version" blah
- redefine ACMRed to give a less visually distracting colour for links.
- redefine the use of \textrightarrow in the CCS to avoid some problem with
T1 encoding and textcomp

best,
Peter

**Comment by borisveytsman** [comment]
@PeterSewell , I understand these woes.  However, the decision to implement these changes - with one exception - is for ACM to make.

Exception:  can you please explain your problem with $\textrightarrow$?

**Comment by PeterSewell** [comment]
On 12 November 2017 at 19:15, Boris Veytsman <notifications@github.com>
wrote:

> @PeterSewell <https://github.com/petersewell> , I understand these woes.
> However, the decision to implement these changes - with one exception - is
> for ACM to make.
>
> Exception: can you please explain your problem with $\textrightarrow$?
>

with acmart as-is, I got an error:

(/usr/share/texmf-texlive/tex/latex/psnfss/ts1ptm.fd)

! Package textcomp Error: Symbol \textrightarrow not provided by
(textcomp)                font family ptm in TS1 encoding.
(textcomp)                Default family used instead.

See the textcomp package documentation for explanation.
Type  H <return>  for immediate help.

\newcommand{\mytextrightarrow}{$\rightarrow$}

IIRC from the arrows in the CCS.

**Comment by borisveytsman** [comment]
Why does your file load `ts1ptm.fd`? It should not.  Could you please post a preamble of your tex file?

Also, please add (before `\documentclass`) the command `\listfiles` and post the list of files from the log.

**Comment by PeterSewell** [comment]
On 12 November 2017 at 21:35, Boris Veytsman <notifications@github.com>
wrote:

> Why does your file load ts1ptm.fd?
>
no idea


> It should not. Could you please post a preamble of your tex file?
>
> Also, please add (before \documentclass) the command \listfiles and post
> the list of files from the log.
>

attached

best,
Peter

**Comment by borisveytsman** [comment]
I do not see the attachment :(

**Comment by PeterSewell** [comment]
they were there - I guess the reply.github.com is stripping them.  another
email?

**Comment by borisveytsman** [comment]
Maybe you could attach them using the web interface?  Try https://github.com/borisveytsman/acmart/issues/169

**Comment by PeterSewell** [comment]
I don't (now I think of it) really want to make them publicly visible and
archived.

**Comment by borisveytsman** [comment]
ok, send them me at borisv@lk.net

**Comment by borisveytsman** [comment]
Here is the problem:  `\renewcommand{\rmdefault}{ptm}`.  This changes the font of your document.  This leads to many other problems, since `acmart` is designed to use Libertine fonts as required by ACM. 

You also redefine many other `acmart` features, and some changes are not compatible with `acmart`.

If you want to typeset a paper according to ACM rules, you may want to use `acmart`.  If you want to typeset a paper with different fonts and typographical details, you may want to use another class.  Taking `acmart` and redefining its internals is possible, but it is difficult for me to help in this case.  If you buy a Subaru, and install an engine from Honda, you should not expect Subaru experts to easily help you.

The way `acmart` is programmed is caused by the requirements of ACM publications.   It is not intended for other purposes.

**Comment by PeterSewell** [comment]
On 12 November 2017 at 23:56, Boris Veytsman <notifications@github.com>
wrote:

> Here is the problem: \renewcommand{\rmdefault}{ptm}. This changes the
> font of your document. This leads to many other problems, since acmart is
> designed to use Libertine fonts as required by ACM.
>
thanks - I wasn't aware that was in the document


> You also redefine many other acmart features, and some changes are not
> compatible with acmart.
>
what did you notice?


> If you want to typeset a paper according to ACM rules, you may want to use
> acmart. If you want to typeset a paper with different fonts and
> typographical details, you may want to use another class. Taking acmart
> and redefining its internals is possible, but it is difficult for me to
> help in this case. If you buy a Subaru, and install an engine from Honda,
> you should not expect Subaru experts to easily help you.
>
> The way acmart is programmed is caused by the requirements of ACM
> publications. It is not intended for other purposes.
>
well, this draft _is_ intended for an ACM publication - but it adapts
content from a number of previous papers, typeset with earlier ACM and
other styles, and with some auto-generated code.  So I did the minimal
get-it-to-compile-and-look-reasonable changes to port to this acmart

thanks again,
Peter

**Comment by Craig-Rodkin** [comment]
Hi,

We will take this issue up with the volunteers with the end of year template review.

Regards,

-Craig

Craig Rodkin
Publications Operations Manager
Association for Computing Machinery
rodkin@hq.acm.org<mailto:rodkin@hq.acm.org>
http://member.acm.org/~rodkin

Explore Association for Computing Machinery's AUTHOR-IZER SERVICE:  http://www.acm.org/publications/acm-author-izer-service<http://www.acm.org/publications/acm-author-izer-service>

**Comment by borisveytsman** [comment]
@PeterSewell 

> what did you notice?

Well, you add `ae` fonts.  You change lists environments.  You have customized raised citations and refs.  You change headers.  You redefine proof.

It is important that journals and proceedings have uniform look and feel.  The changes like these work against it.  Some editors would just reject a manuscript which cannot me made conforming with the journal or proceedings style.

Because of problems like these some journals now refuse to accept LaTeX manuscripts and take Word files - even if they rekey them in TeX in their back offices.  It just takes too much expert time to strip the manuscript of authors' customization code.

**Comment by borisveytsman** [comment]
@Craig-Rodkin I would be happy to implement these changes

**Comment by PeterSewell** [comment]
On 13 November 2017 at 15:18, Craig-Rodkin <notifications@github.com> wrote:

> Hi,
>
> We will take this issue up with the volunteers with the end of year
> template review.
>

thanks

**Comment by vsahil** [comment]
This worked for me. FYI: I just read the first comment in this thread. 
`\settopmatter{printacmref=false}`
`\setcopyright{none}`
`\renewcommand\footnotetextcopyrightpermission[1]{}`
`\pagestyle{plain}`

**Comment by kargaranamir** [comment]
For the new folks: 
To remove copyright box, just add nonacm=true, authorversion=true:
```
% \documentclass[sigconf, authorversion=true, nonacm=true]{acmart}
```

**Comment by xicocaio** [comment]
Hi @kargaranamir,

In the case of uploading the pre-print of an accepted paper to arXiv, using `authorversion=true, nonacm=true` removes the DOI, and according to the [author rights](https://authors.acm.org/author-resources/author-rights)

> **ACM does request authors**, who post to ArXiv or other permitted sites, **to also post the published version's Digital Object Identifier (DOI) alongside the pre-published version on these sites**, so that easy access may be facilitated to the published "Version of Record" upon publication in the ACM Digital Library.

One option would be to use only `authorversion=true`, but then the copyright strip would be present.

Any ideas on how to deal with this?

**Comment by kargaranamir** [comment]
Hi @xicocaio,
As I realized, the author rights could be satisfied with putting the DOI on the arxiv page of your paper, not necessarily in your paper. 
See this [example page](https://arxiv.org/abs/2004.14826), search `DOI`.

**Comment by xicocaio** [comment]
Hi @kargaranamir, thank you very much for the help.

PS: Just out of curiosity about your example regarding the pre-print you added to arXiv. Did you upload the accepted version that includes adjustments following reviewers' suggestions?

**Comment by kargaranamir** [comment]
Your welcome @xicocaio,
Yes 👍.

**Comment by borisveytsman** [comment]
The `nonacm` mode solves this.
